x/tally/module.go

```diff
@@ -144,5 +145,13 @@ func (am AppModule) BeginBlock(_ context.Context) error {
 
 // EndBlock returns the end block logic for the tally module.
 func (am AppModule) EndBlock(ctx context.Context) error {
+	start := telemetry.Now()
+	defer telemetry.ModuleMeasureSince(types.ModuleName, start, "seda_tally_end_block_time")
+
+	// Set this to zero so it always shows up even if there are no tallies
+	telemetry.SetGauge(0.0, "seda_tally_end_block_number_of_data_requests_to_tally")
+	// We reset the amount of gas burned in the end block
+	// since we only add to it during the processing of tallies
+	telemetry.SetGauge(0.0, "seda_tally_end_block_tera_gas_burned")
```

Comment:
Should we create consts for these two?

Reply:
I think HY meant consts for the labels? That makes sense to me as we’re using them in multiple places

Reply:
Ah ok for the strings I'll add consts